src/page-modules/assistant/layout.tsx

```diff
@@ -81,11 +82,12 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
           transition={{ duration: 0.25, ease: [0.04, 0.62, 0.23, 0.98] }}
           className={style.main}
         >
+          <AssistantDepartureToggleButton />
+
           <div className={style.input}>
             <Typo.p textType="body__primary--bold" className={style.heading}>
               {t(PageText.Assistant.search.input.label)}
             </Typo.p>
-
```

Comment:
Have you set up some auto-formatting that removes these lines? 

We use Prettier in this project, so if you use e.g. VS Code you should configure format on save to use the Prettier config.

Reply:
Thanks! For some reason .prettier was not configured in my vs code. Configured it back now.

Reply:
Great! Maybe we can add those deleted blank lines back?